src/oralb_ble/parser.py

```diff
@@ -225,10 +249,16 @@ def _start_update(self, service_info: BluetoothServiceInfo) -> None:
         tb_state = STATES.get(state, f"unknown state {state}")
         tb_mode = modes.get(mode, f"unknown mode {mode}")
         tb_pressure = PRESSURE.get(pressure, f"unknown pressure {pressure}")
-        tb_sector = SECTOR_MAP.get(sector, f"sector {sector}")
+        tb_sector = SECTOR_MAP.get(sector, f"unknown sector code {sector}")
 
         self.update_sensor(str(OralBSensor.TIME), None, time, None, "Time")
-        self.update_sensor(str(OralBSensor.SECTOR), None, tb_sector, None, "Sector")
+        if time == 0 and (state == 2 or state == 8):
```

Comment:
Can you make 2 and 8 constants so we know what they are when we look back on this in the future?

Reply:
can do

Reply:
changed it to tb_state != "running" as looking at the states again, we don't care about sector for any of them outside of running, so I figure this would be better at preventing any kind of bugs that may pop up in the future